awx/main/management/commands/create_preload_data.py

```diff
@@ -23,44 +23,50 @@ def handle(self, *args, **kwargs):
         with impersonate(superuser):
             with disable_computed_fields():
                 if not Organization.objects.exists():
-                    o = Organization.objects.create(name='Default')
+                    o, _ = Organization.objects.get_or_create(name='Default')
 
-                    p = Project(
+                    p, _ = Project.objects.get_or_create(
                         name='Demo Project',
                         scm_type='git',
                         scm_url='https://github.com/ansible/ansible-tower-samples',
                         scm_update_on_launch=True,
```

Comment:
Personally, I frequently go into the UI and disable `scm_update_on_launch` for the demo project. If I read this right, that would cause subsequent runs of this script to error. Because of my change, a project with these attributes won't exist, so between get & create, it will create, but a project with the (name, organization) identity tuple does exist, thus, probably an error.

So I would prefer to eventually move to the documented solution that the non-identity fields get passed in the `defaults` kwarg. For an example

https://github.com/ansible/awx/blob/2afa406b7f16971ea83150c326f8e04717b5c14d/awx/main/management/commands/register_default_execution_environments.py#L77-L82